about/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Об авторе</title>
+</head>
+<body>
+    <h1>&#9996; Немного обо мне</h1>
+    <p>Меня зовут Оксана. Мне на данный момент 2<sup>1</sup> год. Родилась в небольшом городке и большую часть жизни прожила там. <br/>После школы я переехала в Минск и начала поиски себя. <strong>Мой девиз по жизни</strong>:</p>
+    <blockquote>
+        <p><em>Если вы хотите сделать что-то великое в один прекрасный день,<br/> помните: один прекрасный день — это сегодня.</em></p>
+    </blockquote>
+    <hr/>
+    <h2>&#128099; Почему я выбрала Frontend разработку?</h2>
+    <p>Так как я уже <mark>более двух лет</mark> работаю менеджером в продуктовой IT-компании, очень часто приходится принимать участие в<br/> разработке нового продукта. Хочу углубить свои знания в этом напралении и быть полезной в этой сфере. </p>
+<hr/>
+<h3>&#128170; Мои сильные качества разработчика</h3>
+<p>Умение работать в <ins>любой</ins> обстановке</p>
+<p>Готовность к исследованиям</p>
+<p>Коммуникационные навыки</p>
+<p>Способность планировать</p>
+<hr/>
+<p><a href="mailto:liseczkaya@gmail.com" target="_blank">Моя почта</a></p>
+<p><a href="https://github.com/oksana-liseczkaya" target="_blank">Мой профиль в GitHub</a></p>
```

Comment:
Please add indent from 18 to 26 line. It will be more prettier.

Reply:
add indent from 18 to 26 line